src/ast2ram/Location.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file Location.h
+ *
+ ***********************************************************************/
+
+#pragma once
+
+#include "souffle/utility/ContainerUtil.h"
+#include "souffle/utility/MiscUtil.h"
+
+namespace souffle::ram {
+class RelationReference;
+}
+
+namespace souffle::ast2ram {
+
+struct Location {
+    int identifier{};
```

Comment:
Does a default make sense? If it's feasible, it might be better to remove the default constructor. Can they also be const, or do they need to be mutable?